### PR TITLE
Added more sensors for Exalt ExtendAir devices

### PR DESCRIPTION
**Pull request by laf**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

### CODE REVIEW

File: includes/discovery/sensors/state/extendair.inc.php [hunk 1]
```diff
@@ -131,6 +204,78 @@
...
+$cur_oid = '.1.3.6.1.4.1.25651.1.2.4.2.3.1.15.0';
+
+if (is_numeric($temp)) {
+    $state_name = 'locLinkStateV';
+    $index = $state_name;
+    $state_index_id = create_state_index($state_name);
+
+    if ($state_index_id !== null) {
+        $states = array(
+            array($state_index_id,'almNORMAL',1,0,0) ,
```
Comment (Rosiak) [inline includes/discovery/sensors/state/extendair.inc.php]: Wouldn't it be worth to consolidate the state translations, in this case, as they practically all are the same?

Comment (laf) [inline includes/discovery/sensors/state/extendair.inc.php]: You mean one state name?

**Comment by esorin0** [comment]
https://pastebin.com/c71afa8x

**Comment by laf** [comment]
According to that output, the two new sensors have been detected. What's actually wrong for you?

**Comment by esorin0** [comment]
I have just posted the output, nothing is wrong.
You've added Frequency (MHz), which is something that is not really changing for a radio of this type and it's not really useful. Same with 'Link security status' state, which is a constant for a given link. Can we show TX/RX power on the device main page? Thanks!

**Pull request merged**